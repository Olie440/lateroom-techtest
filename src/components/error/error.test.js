import React from 'react';
import Error from './error.component';
import { shallow } from 'enzyme';

describe('<Error />', () => {
    it('renders correctly', () => {
        const component = shallow(<Error />);
        expect(component).toMatchSnapshot();
    });

    it('displays the passed in message', () => {
        const component = shallow(<Error message="testestets" />);
        expect(component).toMatchSnapshot();
    });
});
