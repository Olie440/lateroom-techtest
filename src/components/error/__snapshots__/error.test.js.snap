// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> displays the passed in message 1`] = `
<div
  className="error"
>
  <p
    className="error__title"
  >
    Error
  </p>
  <p
    className="error__body"
  >
    testestets
  </p>
</div>
`;

exports[`<Error /> renders correctly 1`] = `
<div
  className="error"
>
  <p
    className="error__title"
  >
    Error
  </p>
  <p
    className="error__body"
  />
</div>
`;
